src/js/src/components/SearchSingleItemComponents/SearchSingleItemAllData.vue

```diff
@@ -2,7 +2,7 @@
   <div class="SingleEntryAllData">
     <div class="showAllButtonContainer">
       <button class="allDataButton" @click="toggleAllDataShown">
-        See all data
+        {{ allDataButtonText }}
```

Comment:
Consider using a computed property for the text toggling and then just toggle the `allDataShown` property above directly in the click handler. Gives a bit cleaner code in this case.

Reply:
Fixed.